gnc/sonar/src/BVWrapper.cpp

```diff
@@ -0,0 +1,175 @@
+#include "BVWrapper.h"
+
+namespace bv{
+	bool BVWrapper::mapColors(BVTMagImage img, BVTColorImage& cimg){
+		if( this->mapper == NULL )
+		{
+			printf("BVTColorMapper_Create: failed\n");
+			return false;
+		}
+		
+		// Load the bone colormap
+		int ret = BVTColorMapper_Load(this->mapper, (ros::package::getPath("sonar") + "/res/jet.cmap").c_str());
```

Comment:
The path could also be a rosparam.
I think this package should be called `sub8_imaging_sonar`, since `sonar` is ambiguous (since we currently have like two other sonar packages)